pages/index.vue

```diff
@@ -1835,6 +1837,14 @@ export default {
     background linear-gradient(to bottom, rgb(0,0,0) 30%, rgba(0,0,0,0) 100%)
     filter saturate(180%)
 
+// TODO: temp fix for safari sticky bar
+@supports (position: -webkit-sticky)
```

Comment:
This isn't fixing things in Safari – the page nav is now `fixed` to the top of the page. I thought this is what Headroom was supposed to solve with good old fashioned JS scroll watching?